lib/MultipartDataGenerator.js

```diff
@@ -39,7 +39,10 @@ function multipartDataGenerator(method, data, headers) {
   }
   push('--' + segno + '--');
 
-  return buffer;
+  if (!callback) {
```

Comment:
This function doesn't have any async pieces; does it really need the callback in here?

Reply:
you're correct - definitely does not need to have a callback in there. I'll update. thanks @jlomas-stripe